docs/developer-guide/release.md

```diff
@@ -22,55 +22,86 @@ Add an entry in the changelog like this:
-    - [ ] Commit on master 0.&lt;x-incremented&gt;.0
-    - [ ] Make sure on YYYY.XX.xx we have the 0.x.&lt;number-of-minor-version&gt;
+- [ ] If major release, Change [QA Jenkins job](http://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2-QA-Build/) to build the new branch, enable the job continuous deploy by updating the `branch` parameter in the build configuration page to `YYYY.XX.xx`
+- [ ] Check version in `package.json`. (as for semantic versioning the major have to be 0 until the npm package has not a stable API).
+    - [ ] Take note of current version of mapstore in `package.json` in master branch, it should be in the form 0.x.0
+    - [ ] If major release, make pr and merge on master **0.&lt;x-incremented&gt;.0**
+    - [ ] if minor release, make pr and merge on stable YYYY.XX.xx **0.x.&lt;number-of-minor-version&gt;**
 - [ ] Create a milestone on GitHub with the same name of the release (vYYYY.XX.xx)
-    - [ ] create a temporary label "current release" and assing the label to all the issues of the current zenhub release
-    - [ ] use the label to filter the issues on github and assign to all the issue the milestone created
-    - [ ] remove the temporary label "current release"
-- [ ] Update `CHANGELOG.md`. [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation) - both master and stable
+    - [ ] assign the label "current-release" to all the issues and Prs of the current zenhub release
+    - [ ] use the label to filter the issues on github and assign to all the issues and Prs the milestone created
+    - [ ] remove assignments of "current-release" 
+- [ ] Prepare PR for updating `CHANGELOG.md` for **master** and **stable** [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation)
 - [ ] Fix `pom.xml` dependencies stable versions ( no `-SNAPSHOT` usage release).
 - [ ] Update the version of java modules on the stable branch to a stable, incremental version.
-    - [ ] Run `mvn versions:set -DnewVersion=<VERSION> -DprocessAllModules -DgenerateBackupPoms=false` to update package version, where <VERSION> is the version of the java packages (e.g. `1.2.2`).
+    - [ ] Run `mvn release:update-versions -DdevelopmentVersion=<version> -Pprinting,printingbundle,release
+` to update package version, where <VERSION> is the version of the java packages (e.g. `1.2.2`).
     - [ ] Manually update project pom templates to use `mapstore-services` of `<VERSION>`
-- [ ] Release a stable `mapstore-services`. (from `2022.01.xx` also mapstore-webapp should be deployed for new project system and product).
+- [ ] Release a stable `mapstore-services`. (from `2022.01.xx` also mapstore-webapp (java/web) should be deployed for new project system and product). 
+  - [ ] for mapstore-webapp use `mvn clean install deploy -f java/pom.xml`
+  - [ ] for the product use `mvn clean install deploy -f product/pom.xml` if this does not work it can be ignored
```

Comment:
Thinking well, I should not deploy the war at this stage, we may have didn't compiled JS part so we may publish something not well done. Let's remove this step.
```suggestion
```